2023-01-06 19:54:46,703:INFO:__main__:Skaiciu (4, 5, 7, 8, 9, 9) suma lygi: 42
2023-01-06 19:54:46,703:ERROR:__main__:Saknis gali buti traukiama tik is skaiciaus
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 21, in saknis
    rezultatas = math.sqrt(x)
                 ^^^^^^^^^^^^
TypeError: must be real number, not str
2023-01-06 19:54:46,704:INFO:__main__:Sakinio Labas vakaras ilgis lygus 13 simboliu
2023-01-06 19:54:46,704:ERROR:__main__:Dalyba is nulio
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 34, in dalyba
    rezultatas = x / y
                 ~~^~~
ZeroDivisionError: division by zero
2023-01-06 19:55:24,133:INFO:__main__:Skaiciu (4, 5, 7, 8, 9, 9) suma lygi: 42
2023-01-06 19:55:24,134:ERROR:__main__:Saknis gali buti traukiama tik is skaiciaus
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 21, in saknis
    rezultatas = math.sqrt(x)
                 ^^^^^^^^^^^^
TypeError: must be real number, not str
2023-01-06 19:55:24,135:INFO:__main__:Sakinio Labas vakaras ilgis lygus 13 simboliu
2023-01-06 19:55:24,135:ERROR:__main__:Dalyba is nulio
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 34, in dalyba
    rezultatas = x / y
                 ~~^~~
ZeroDivisionError: division by zero
2023-01-06 19:55:32,235:INFO:__main__:Skaiciu (4, 5, 7, 8, 9, 9) suma lygi: 42
2023-01-06 19:55:32,235:ERROR:__main__:Saknis gali buti traukiama tik is skaiciaus
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 21, in saknis
    rezultatas = math.sqrt(x)
                 ^^^^^^^^^^^^
TypeError: must be real number, not str
2023-01-06 19:55:32,236:INFO:__main__:Sakinio Labas vakaras ilgis lygus 13 simboliu
2023-01-06 19:55:32,236:INFO:__main__:10 padalinta is 12 lygu 0.8333333333333334
2023-01-06 19:55:49,704:INFO:__main__:Skaiciu (4, 5, 7, 8, 9, 9) suma lygi: 42
2023-01-06 19:55:49,704:ERROR:__main__:Saknis gali buti traukiama tik is skaiciaus
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 21, in saknis
    rezultatas = math.sqrt(x)
                 ^^^^^^^^^^^^
TypeError: must be real number, not str
2023-01-06 19:55:49,706:INFO:__main__:Sakinio Labas vakaras ilgis lygus 13 simboliu
2023-01-06 19:55:49,706:ERROR:__main__:Dalyba is nulio
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 34, in dalyba
    rezultatas = x / y
                 ~~^~~
ZeroDivisionError: division by zero
2023-01-06 19:56:35,400:INFO:__main__:Skaiciu (4, 5, 7, 8, 9, 9) suma lygi: 42
2023-01-06 19:56:35,400:ERROR:__main__:Saknis gali buti traukiama tik is skaiciaus
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 24, in saknis
    rezultatas = math.sqrt(x)
                 ^^^^^^^^^^^^
TypeError: must be real number, not str
2023-01-06 19:56:35,402:INFO:__main__:Sakinio Labas vakaras ilgis lygus 13 simboliu
2023-01-06 19:56:35,402:ERROR:__main__:Dalyba is nulio
Traceback (most recent call last):
  File "C:\Users\<user name>\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch_31.py", line 37, in dalyba
    rezultatas = x / y
                 ~~^~~
ZeroDivisionError: division by zero
